.travis.yml

```diff
@@ -26,6 +26,11 @@ addons:
             - graphviz
             - language-pack-de
 
+# This should allow building on arm64
+arch:
```

Comment:
Remove this section, listing the `arch` in the jobs should be enough

Reply:
all right. I'm trying to follow https://blog.travis-ci.com/2019-10-07-multi-cpu-architecture-support

Reply:
Still, the result of this section is to add job no3 and no4, neither of those are needed.

Reply:
(we don't have any jobs in the default matrix, but it's just triggering that empty one nevertheless)